libmobilecoin/src/transaction.rs

```diff
@@ -481,6 +481,7 @@ pub extern "C" fn mc_transaction_builder_add_output(
     recipient_address: FfiRefPtr<McPublicAddress>,
     rng_callback: FfiOptMutPtr<McRngCallback>,
     out_tx_out_confirmation_number: FfiMutPtr<McMutableBuffer>,
+    out_tx_out_shared_secret: FfiMutPtr<McMutableBuffer>,
```

Comment:
Since we're switching `TransactionBuilder::add_*output` to return `TxOutContext`, would it make sense to similarly make these FFI `add_output` methods return the whole `TxOutContext`, replacing these 2 out pointers?

Reply:
yes, using the second out pointer was my idea, but I think returning a whole `TxOutContext` would be better.

Reply:
Ill work on a commit to make this change